### PR TITLE
#5-iphone-screens

### PR DESCRIPTION
**Pull request by Dawindows**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -124,6 +126,66 @@
...
+        </div>
+        <div class="iphone-slider">
+        <div class="iphone-screen">
+            <div class="content"><img src="assets/img/screen-1.jpg" alt=""></div>
+            <div class="content"><img src="assets/img/screen-2.jpg" alt=""></div>
+            <div class="content1"><img src="assets/img/iphone-6.png" alt=""></div>
+            <div class="content"><img src="assets/img/screen-3.jpg" alt=""></div>
+            <div class="content"><img src="assets/img/screen-4.jpg" alt=""></div>
+        </div>
+        </div>
```
Comment (Dawindows) [inline index.html]: I don't know how to fix it.

File: index.html [hunk 2]
```diff
@@ -88,10 +88,12 @@
                     <p>We thought - Mr.Copperfield thought - it was quite a large rookery;but the nests were very old ones, and the birds have them a long while.
                     </p>
                 </div>
-                <div class="button-2">
+                <div class="iphone-feature-button">
                     <button class="button-2">Try to hack us <i class="fas fa-arrow-right"></i></button>
+                    <button class="button-3">Learn more</button>
```
Comment (KatyaGunko) [inline index.html]: please rename those classes

File: index.html (hunk 1 above)
Comment (KatyaGunko) [reply index.html]: What type of problem?

File: index.html (hunk 2 above)
Comment (Dawindows) [reply index.html]: > please rename those classes

renamed)

File: src/style.css [hunk 3]
```diff
@@ -278,39 +279,42 @@ li {
...
 	border-radius: 30px;
 	border: inherit;
 	font-size: 16px;
 	color: #fff;
 	background: #257df4;
 }
 
-.button-2 i{
+.button-hack i {
 	margin-left: 41px;
```
Comment (KatyaGunko) [inline src/style.css]: margin is no supposed to be 41px

File: src/style.css [hunk 4]
```diff
@@ -356,20 +379,14 @@ li {
 		font-size: 15px;
 	}
 
-}
+	.iphone-feature-button {
+		margin-top: 45px;
```
Comment (KatyaGunko) [inline src/style.css]: margin is not supposed to be 45px

**Pull request merged**